server/lib/budget.js

```diff
@@ -107,6 +107,42 @@ export function getTotalAmountReceivedAmount(collective, { startDate, endDate, c
   });
 }
 
+export async function getTotalAmountPaidExpenses(collective, { startDate, endDate, expenseType, currency } = {}) {
+  currency = currency || collective.currency;
+
+  const where = {
+    FromCollectiveId: collective.id,
+    status: 'PAID',
+  };
+  if (expenseType) {
+    where.type = expenseType;
+  }
+  if (startDate) {
+    where.createdAt = where.createdAt || {};
+    where.createdAt[Op.gte] = startDate;
+  }
+  if (endDate) {
+    where.createdAt = where.createdAt || {};
+    where.createdAt[Op.lt] = endDate;
+  }
+
+  const results = await models.Expense.findAll({
+    attributes: ['currency', [sequelize.fn('COALESCE', sequelize.fn('SUM', sequelize.col('amount')), 0), 'amount']],
+    where: where,
+    group: 'currency',
+    raw: true,
+  });
+
+  let total = 0;
+  for (const result of Object.values(results)) {
```

Comment:
Don't see the point of `Object.values(results)`, `results is an array and we should just use it like this: `results`